libpkg/clean_cache.c

```diff
@@ -48,7 +48,7 @@ rm_rf(int basefd, const char *path)
 
 		dirfd = openat(basefd, path, O_DIRECTORY|O_CLOEXEC);
 		if (dirfd == -1) {
-			pkg_emit_errno("openat", path);
+			pkg_emit_errno("openat", __func__);
```

Comment:
It was more of a case of standardising on the way the function should be called.  In talking with @bapt that's what we agreed on.  I don't see this being any less of an exception -- it's a lot easier to see which originating function.